purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -354,25 +354,34 @@ class Purchases internal constructor(
         }
     }
 
+    /**
+     * Gets the StoreProduct(s) for the given list of product ids for all product types.
+     * @param [productIds] List of productIds
+     * @param [callback] Response callback
+     */
+    fun getProducts(
+        productIds: List<String>,
+        callback: GetStoreProductsCallback
+    ) {
+        getProducts(productIds, null, callback)
+    }
+
     /**
      * Gets the StoreProduct(s) for the given list of product ids.
      * @param [productIds] List of productIds
+     * @param [type] A product type to filter by
      * @param [callback] Response callback
      */
     fun getProducts(
         productIds: List<String>,
+        type: ProductType? = null,
```

Comment:
can we just have the one function with the optional parameter?

Reply:
@beylmk I tried to do just one but then like the developer needs to pass a null which feels gross like `getProducts(ids, null, callback)` which feels gross 😔

Reply:
spoke with josh on slack and we are going to keep both, omitting requires named parameters...